v23.1/build-a-python-app-with-cockroachdb-asyncpg.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Build a Python App with CockroachDB and asyncpg
+summary: Learn how to use CockroachDB from a simple Python application with the asyncpg driver.
+toc: true
+twitter: false
+referral_id: docs_python_asyncpg
+filter_category: crud_python
+filter_html: <strong>asyncpg</strong>
+filter_sort: 1
+docs_area: get_started
+---
+
+{% include {{ page.version.version }}/filter-tabs/crud-python.md %}
+
+{% include cockroach_u_pydev.md %}
+
+This tutorial shows you how build a simple Python application with CockroachDB and the [asyncpg](https://magicstack.github.io/asyncpg/current/index.html) driver.
+
+## Step 1. Start CockroachDB
+
+{% include {{ page.version.version }}/setup/sample-setup-certs.md %}
+
+## Step 2. Get the sample code
+
+Clone the sample code's GitHub repo:
+
+{% comment %}
```

Comment:
Does this need to be updated before merging?

Reply:
Yup, will remove the comments.